bin/extract-release-notes.sh

```diff
@@ -19,4 +19,4 @@ VERSION_PATTERN="^## v"
 result=$(awk "/$VERSION_PATTERN/{c++; if(c==2) exit;} c==1" "$CHANGELOG_FILE")
 
 # Print the result
-echo "$result" |  grep -v "^#"
+echo "$result" |  grep -v "$VERSION_PATTERN"
```

Comment:
This just keeps strings like `### added` `### changed` to be more [keepachangelog.com like](https://keepachangelog.com/en/1.1.0/)